Civi/Funding/PayoutProcess/Validator/DrawdownValidator.php

```diff
@@ -63,6 +67,27 @@ public function __construct(FundingCaseManager $fundingCaseManager, PayoutProces
    * phpcs:disable Drupal.Commenting.FunctionComment.IncorrectTypeHint
    */
   public function validate(AbstractEntity $new, AbstractEntity $current): EntityValidationResult {
+    $payoutProcess = $this->getPayoutProcess($new);
+    $this->assertNotClosed($payoutProcess);
+
+    if ($new->getAmount() < 0) {
+      $payoutProcess = $this->getPayoutProcess($new);
+      $fundingCase = $this->getFundingCase($payoutProcess);
+
+      if (!$this->hasReviewPermission($fundingCase->getPermissions())) {
+        return $this->createAmountLessThanZeroResult();
+      }
+    }
+
+    if ($new->getAmount() > $current->getAmount()) {
```

Comment:
is it ok that `$new->getAmount()` is negative if line 77 did not enter if-branch?

Reply:
Actually this should not happen, though I don't think it is necessary to prohibit it here.